app/authentication/login/login.component.e2e-spec.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+var protractor_1 = require("protractor");
+describe('Login', function () {
+    beforeEach(function () {
```

Comment:
__[jshint]__
* 'beforeEach' is not defined.

<!-- Comment by [SideCI](https://sideci.com) -->